sailfishusermanager.cpp

```diff
@@ -537,11 +537,12 @@ void SailfishUserManager::onUnitJobFinished(SystemdManager::Job &job)
 }
 
 void SailfishUserManager::onUnitJobFailed(SystemdManager::Job &job, SystemdManager::JobList &remaining) {
-    emit currentUserChangeFailed(m_switchUser);
     if (job.type == SystemdManager::StopJob && job.unit == USER_SERVICE.arg(m_currentUid)) {
         // session systemd is fubar, autologin is probably still up
         qCWarning(lcSUM) << "Unit failed while stopping session, trying to continue";
         m_systemd->addUnitJobs(remaining); // Try to continue anyway
+        // Inform UI
+        emit currentUserChangeFailed(m_switchUser);
     } else if (job.type == SystemdManager::StopJob && job.unit == AUTOLOGIN_SERVICE.arg(m_currentUid)) {
```

Comment:
Why this block doesn't emit currentUserChangeFailed?

Reply:
If just this autologin stop fails but continue works then the change does not actually fail? Didn't manage to actually test this.

Reply:
Ah okay yes, then the one above should emit it either.